datahub-web-react/src/app/entity/dataset/profile/schema/Schema.tsx

```diff
@@ -208,21 +210,24 @@ export default function SchemaView({ urn, schema, editableSchemaMetadata, update
         );
     };
 
-    const tagGroupRender = (tags: GlobalTags, record: SchemaField, rowIndex: number | undefined) => {
+    const tagAndTermRender = (tags: GlobalTags, record: SchemaField, rowIndex: number | undefined) => {
         const relevantEditableFieldInfo = editableSchemaMetadata?.editableSchemaFieldInfo.find(
             (candidateEditableFieldInfo) => candidateEditableFieldInfo.fieldPath === record.fieldPath,
         );
         return (
-            <TagGroup
-                uneditableTags={tags}
-                editableTags={relevantEditableFieldInfo?.globalTags}
-                canRemove
-                canAdd={tagHoveredIndex === `${record.fieldPath}-${rowIndex}`}
-                onOpenModal={() => setTagHoveredIndex(undefined)}
-                updateTags={(update) =>
-                    onUpdateTags(update, relevantEditableFieldInfo || { fieldPath: record.fieldPath })
-                }
-            />
+            <>
+                <GlossaryTermGroup glossaryTerms={record.glossaryTerms as GlossaryTerms} />
```

Comment:
Since we are always showing tags & terms side by side, instead of making another group with similar layout logic can we render them all in a TagTermGroup?

Reply:
done